engine/src/main/java/org/camunda/bpm/engine/impl/persistence/entity/EverLivingJobEntity.java

```diff
@@ -35,43 +33,21 @@ public class EverLivingJobEntity extends JobEntity {
 
   public static final String TYPE = "ever-living";
 
+  @Override
   public String getType() {
     return TYPE;
   }
 
   @Override
   protected void postExecute(CommandContext commandContext) {
     LOG.debugJobExecuted(this);
-    init(commandContext);
+    init(commandContext, false, true);
     commandContext.getHistoricJobLogManager().fireJobSuccessfulEvent(this);
   }
 
   @Override
   public void init(CommandContext commandContext) {
-    init(commandContext, false);
-  }
-
-  public void init(CommandContext commandContext, boolean shouldResetLock) {
-    // clean additional data related to this job
-    JobHandler jobHandler = getJobHandler();
-    if (jobHandler != null) {
-      jobHandler.onDelete(getJobHandlerConfiguration(), this);
-    }
-
-    //cancel the retries -> will resolve job incident if present
-    int retries = HistoryCleanupHelper.getMaxRetries();
-    setRetries(retries);
-
-    //delete the job's exception byte array and exception message
-    if (exceptionByteArrayId != null) {
-      clearFailedJobException();
-    }
-
-    //clean the lock information
-    if (shouldResetLock) {
-      setLockOwner(null);
-      setLockExpirationTime(null);
-    }
+    init(commandContext, false, false);
```

Comment:
❓Shouldn't this be `false, true`? Before refactoring the delete handler was executed.

Reply:
It did, yes. Actually, it doesn't make any difference as far as I can tell since the `init(CommandContext)` method is never invoked from our code except from this entity itself. But you're right, let's not create a regression by mistake here, I'll adjust this 👍